charts/cluster-api-runtime-extensions-nutanix/addons/nfd/values-template.yaml

```diff
@@ -1,3 +1,5 @@
+image:
+  tag: v0.16.1-minimal
```

Comment:
This is not necessary since nfd 0.15 where it uses a minimal scratch based image by default (also this downgraded the image btw).